app/views/guide_typography.html

```diff
@@ -214,6 +215,23 @@ <h3 class="heading-medium" id="typography-hidden-text">Hidden text (progressive
   </div>
 
 
+  <h3 class="heading-medium" id="typography-class">Use styling automatically</h3>
+  <div class="text">
+    <p>You can use the <code class="code">govuk-prose</code> class around a body of prose. Then certain elements inside will be styled automatically without the need to add specific classes to each of them.</p>
+    <p>That will apply to most typographic elements: headings, lists, links (without govuk_template), code and strong. Tables, paragraphs, links (with govuk_template) and horizontal rules are already always styled without that class.</p>
```

Comment:
I agree that especially the mentioning of govuk_template is unclear. I'm not sure how important the distinction is because I don't know how many services are using Elements without govuk_template.

What about the following?
> That will apply to most typographic elements: headings, lists, links, code and strong. Tables, paragraphs and horizontal rules are already always styled without that class (and if you are using govuk_template, that includes links).

I have used examples of all of those in the snippet, except
* most headings -- I thought just one type of heading (`h3` in this case) is enough and adding more might be distracting
* links -- the app which displays the example snippet includes govuk_template, so there won't be any difference for links if they are within the `govuk-prose` class or not